cards/src/main/resources/cards/custom/group6/witch_doctor/weapon_sacrificial_blade.json

```diff
@@ -10,7 +10,7 @@
   "battlecry": {
     "condition": {

```

Comment:
this should probably be a comparison condition with a `DiedThisTurnValueProvider`